src/data/bundle.lean

```diff
@@ -79,3 +88,38 @@ instance [add_comm_monoid F] [module R F] : module R (bundle.trivial B F b) := 
 end trivial_instances
 
 end bundle
+
+section bundle_sections
+
+/-! ## Section of bundles -/
+
+open bundle
+
+variables {B : Type*} {E : B → Type*}
+
+/-- Type synonym for name clarity. -/
+@[reducible, nolint has_coe_to_fun] def bundle_section (E : B → Type*) := Π x, E x
```

Comment:
What does this `nolint` do?

Reply:
It tells the linter to not complain about the fact that I did not declare a `has_coe_to_fun` instance

Reply:
Are you sure? The linter doesn't complain for me:
```
import tactic.lint

variables {B : Type*} {E : B → Type*}

/-- Type synonym for name clarity. -/
@[reducible] def bundle_section (E : B → Type*) := Π x, E x

#lint  /- All linting checks passed! -/
```